flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/MultipleInputStreamTask.java

```diff
@@ -281,11 +277,4 @@ public void abortCheckpointOnBarrier(long checkpointId, CheckpointException caus
         }
         super.abortCheckpointOnBarrier(checkpointId, cause);
     }
-
-    @Override
-    protected void advanceToEndOfEventTime() throws Exception {
-        for (Output<StreamRecord<?>> sourceOutput : operatorChain.getChainedSourceOutputs()) {
-            sourceOutput.emitWatermark(Watermark.MAX_WATERMARK);
-        }
-    }
```

Comment:
?

Reply:
Fixed in a later commit. It was mistakenly removed.

Reply:
That's a mistake from one of my previous tries...